### DISCUSSION TITLE
What's wrong with this code. It won't start the mower and override the schedule

### DISCUSSION THREAD

**Discussion by janstolte** [Thomas55555/husqvarna_automower#445] [Q&A] [answered]
I've been pulling my hair out (if any...) to create a button that overrides the schedule for 3 hours and starts mowing.
This is the code I'm using in the button:

```
tap_action:
  action: call-service
  service: husqvarna_automower.custom_command
  service_data:
    command_type: actions
    json_string: |-
      { "data": {
        "type": "Start",
        "attributes": {
          "duration": 180
        }
      }
    entity_id: vacuum.otto
```
If I replace the data part in the json_string with ResumeSchedule, the button works just fine and the mower resumes as per the schedule.
For what it's worth: no error messages at all, so it seems to eat the command just fine, but no action unfortunately.

**Comment by Thomas55555** [comment]
What if you write something else the `Start`, e.g. `Dart` - do you get than an error message?
Have you considered this?
`NOTE: If you send a start command, Automower® will not leave the charging station until it is fully charged.`

**Comment by janstolte** [comment]
I replaced `Start` with `Fart` and it did not throw an error.
Strange...

**Reply by Thomas55555** [reply, marked as answer]
This worked for me:
```
service: husqvarna_automower.custom_command
data:
  command_type: actions
  json_string: |-
    { "data": {
          "type": "Start",
          "attributes": {
            "duration": 60
          }
        }
        }
target:
  entity_id: vacuum.haffi
```

You missed a `}`

**Reply by janstolte** [reply]
Poor me... couldn't even count to three twice... ;)

**Comment by Thomas55555** [comment]
Or even easier:
```
service: number.set_value
data:
   value: "180"
target:
   entity_id: number.otto_mow_for
````

**Reply by janstolte** [reply]
Cool! The shorter code, the better
